[2019-12-24 20:10:55] [WARN ] -  [DUBBO] The timeout response finally returned at 2019-12-24 20:10:55.381, response Response [id=1, version=null, status=20, event=false, error=null, result=RpcResult [result=PageRes(pageNum=1, pageSize=10, startRow=0, endRow=10, total=18, pages=2, count=true, list=[PopSchool(id=1, name=湖南大学, coverImg=https://kzych.oss-cn-qingdao.aliyuncs.com/kzych-dir/hndx_01.jpg), PopSchool(id=2, name=中南大学, coverImg=https://kzych.oss-cn-qingdao.aliyuncs.com/kzych-dir/hndx_01.jpg), PopSchool(id=3, name=湘潭大学, coverImg=https://kzych.oss-cn-qingdao.aliyuncs.com/kzych-dir/hndx_01.jpg), PopSchool(id=4, name=湖南农业大学, coverImg=https://kzych.oss-cn-qingdao.aliyuncs.com/kzych-dir/hndx_01.jpg), PopSchool(id=5, name=长沙理工大学, coverImg=https://kzych.oss-cn-qingdao.aliyuncs.com/kzych-dir/hndx_01.jpg), PopSchool(id=6, name=湖南工商大学, coverImg=https://kzych.oss-cn-qingdao.aliyuncs.com/kzych-dir/hndx_01.jpg), PopSchool(id=7, name=湖南理工大学, coverImg=https://kzych.oss-cn-qingdao.aliyuncs.com/kzych-dir/hndx_01.jpg), PopSchool(id=8, name=湖南师范大学, coverImg=https://kzych.oss-cn-qingdao.aliyuncs.com/kzych-dir/hndx_01.jpg), PopSchool(id=9, name=湖南女子大学, coverImg=https://kzych.oss-cn-qingdao.aliyuncs.com/kzych-dir/hndx_01.jpg), PopSchool(id=10, name=湖南工业大学, coverImg=https://kzych.oss-cn-qingdao.aliyuncs.com/kzych-dir/hndx_01.jpg)]), exception=null]], channel: /192.168.71.1:2576 -> /192.168.71.1:20887, dubbo version: 2.8.4, current host: 192.168.71.1
[2019-12-24 20:10:55] [WARN ] -  [DUBBO] Although retry the method findPopSchool in the service com.xqx.ych.small.api.IOrganizationApi was successful by the provider 192.168.71.1:20887, but there have been failed providers [192.168.71.1:20887] (1/1) from the registry 192.168.71.13:2181 on the consumer 192.168.71.1 using the dubbo version 2.8.4. Last error is: Invoke remote method timeout. method: findPopSchool, provider: dubbo://192.168.71.1:20887/com.xqx.ych.small.api.IOrganizationApi?anyhost=true&application=UserController&check=false&dubbo=2.8.4&generic=false&interface=com.xqx.ych.small.api.IOrganizationApi&methods=findByOrgLevel,findOrgDynamic,findPopSchool,findByMajorId,findOrgIntroduce,findByIdAndMajor,findByOrgId&pid=16524&revision=1.0-SNAPSHOT&side=consumer&timestamp=1577189452842, cause: Waiting server-side response timeout by scan timer. start time: 2019-12-24 20:10:53.921, end time: 2019-12-24 20:10:54.943, client elapsed: 42 ms, server elapsed: 979 ms, timeout: 1000 ms, request: Request [id=1, version=2.0.0, twoway=true, event=false, broken=false, data=RpcInvocation [methodName=findPopSchool, parameterTypes=[int, int, class java.lang.Integer], arguments=[1, 10, 1], attachments={path=com.xqx.ych.small.api.IOrganizationApi, interface=com.xqx.ych.small.api.IOrganizationApi, version=0.0.0}]], channel: /192.168.71.1:2576 -> /192.168.71.1:20887, dubbo version: 2.8.4, current host: 192.168.71.1
com.alibaba.dubbo.rpc.RpcException: Invoke remote method timeout. method: findPopSchool, provider: dubbo://192.168.71.1:20887/com.xqx.ych.small.api.IOrganizationApi?anyhost=true&application=UserController&check=false&dubbo=2.8.4&generic=false&interface=com.xqx.ych.small.api.IOrganizationApi&methods=findByOrgLevel,findOrgDynamic,findPopSchool,findByMajorId,findOrgIntroduce,findByIdAndMajor,findByOrgId&pid=16524&revision=1.0-SNAPSHOT&side=consumer&timestamp=1577189452842, cause: Waiting server-side response timeout by scan timer. start time: 2019-12-24 20:10:53.921, end time: 2019-12-24 20:10:54.943, client elapsed: 42 ms, server elapsed: 979 ms, timeout: 1000 ms, request: Request [id=1, version=2.0.0, twoway=true, event=false, broken=false, data=RpcInvocation [methodName=findPopSchool, parameterTypes=[int, int, class java.lang.Integer], arguments=[1, 10, 1], attachments={path=com.xqx.ych.small.api.IOrganizationApi, interface=com.xqx.ych.small.api.IOrganizationApi, version=0.0.0}]], channel: /192.168.71.1:2576 -> /192.168.71.1:20887
	at com.alibaba.dubbo.rpc.protocol.dubbo.DubboInvoker.doInvoke(DubboInvoker.java:99)
	at com.alibaba.dubbo.rpc.protocol.AbstractInvoker.invoke(AbstractInvoker.java:144)
	at com.alibaba.dubbo.rpc.listener.ListenerInvokerWrapper.invoke(ListenerInvokerWrapper.java:74)
	at com.alibaba.dubbo.monitor.support.MonitorFilter.invoke(MonitorFilter.java:75)
	at com.alibaba.dubbo.rpc.protocol.ProtocolFilterWrapper$1.invoke(ProtocolFilterWrapper.java:91)
	at com.alibaba.dubbo.rpc.protocol.dubbo.filter.FutureFilter.invoke(FutureFilter.java:53)
	at com.alibaba.dubbo.rpc.protocol.ProtocolFilterWrapper$1.invoke(ProtocolFilterWrapper.java:91)
	at com.alibaba.dubbo.rpc.filter.ConsumerContextFilter.invoke(ConsumerContextFilter.java:48)
	at com.alibaba.dubbo.rpc.protocol.ProtocolFilterWrapper$1.invoke(ProtocolFilterWrapper.java:91)
	at com.alibaba.dubbo.rpc.protocol.InvokerWrapper.invoke(InvokerWrapper.java:53)
	at com.alibaba.dubbo.rpc.cluster.support.FailoverClusterInvoker.doInvoke(FailoverClusterInvoker.java:77)
	at com.alibaba.dubbo.rpc.cluster.support.AbstractClusterInvoker.invoke(AbstractClusterInvoker.java:227)
	at com.alibaba.dubbo.rpc.cluster.support.wrapper.MockClusterInvoker.invoke(MockClusterInvoker.java:72)
	at com.alibaba.dubbo.rpc.proxy.InvokerInvocationHandler.invoke(InvokerInvocationHandler.java:52)
	at com.alibaba.dubbo.common.bytecode.proxy3.findPopSchool(proxy3.java)
	at com.xqx.small.web.OrganizationController.findPopSchool(OrganizationController.java:51)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.springframework.web.method.support.InvocableHandlerMethod.doInvoke(InvocableHandlerMethod.java:209)
	at org.springframework.web.method.support.InvocableHandlerMethod.invokeForRequest(InvocableHandlerMethod.java:136)
	at org.springframework.web.servlet.mvc.method.annotation.ServletInvocableHandlerMethod.invokeAndHandle(ServletInvocableHandlerMethod.java:102)
	at org.springframework.web.servlet.mvc.method.annotation.RequestMappingHandlerAdapter.invokeHandlerMethod(RequestMappingHandlerAdapter.java:894)
	at org.springframework.web.servlet.mvc.method.annotation.RequestMappingHandlerAdapter.handleInternal(RequestMappingHandlerAdapter.java:800)
	at org.springframework.web.servlet.mvc.method.AbstractHandlerMethodAdapter.handle(AbstractHandlerMethodAdapter.java:87)
	at org.springframework.web.servlet.DispatcherServlet.doDispatch(DispatcherServlet.java:991)
	at org.springframework.web.servlet.DispatcherServlet.doService(DispatcherServlet.java:925)
	at org.springframework.web.servlet.FrameworkServlet.processRequest(FrameworkServlet.java:981)
	at org.springframework.web.servlet.FrameworkServlet.doGet(FrameworkServlet.java:873)
	at javax.servlet.http.HttpServlet.service(HttpServlet.java:621)
	at org.springframework.web.servlet.FrameworkServlet.service(FrameworkServlet.java:858)
	at javax.servlet.http.HttpServlet.service(HttpServlet.java:728)
	at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:305)
	at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:210)
	at org.apache.tomcat.websocket.server.WsFilter.doFilter(WsFilter.java:51)
	at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:243)
	at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:210)
	at org.springframework.web.filter.CharacterEncodingFilter.doFilterInternal(CharacterEncodingFilter.java:200)
	at org.springframework.web.filter.OncePerRequestFilter.doFilter(OncePerRequestFilter.java:107)
	at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:243)
	at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:210)
	at org.apache.catalina.core.StandardWrapperValve.invoke(StandardWrapperValve.java:222)
	at org.apache.catalina.core.StandardContextValve.invoke(StandardContextValve.java:123)
	at org.apache.catalina.authenticator.AuthenticatorBase.invoke(AuthenticatorBase.java:502)
	at org.apache.catalina.core.StandardHostValve.invoke(StandardHostValve.java:171)
	at org.apache.catalina.valves.ErrorReportValve.invoke(ErrorReportValve.java:100)
	at org.apache.catalina.valves.AccessLogValve.invoke(AccessLogValve.java:953)
	at org.apache.catalina.core.StandardEngineValve.invoke(StandardEngineValve.java:118)
	at org.apache.catalina.connector.CoyoteAdapter.service(CoyoteAdapter.java:408)
	at org.apache.coyote.http11.AbstractHttp11Processor.process(AbstractHttp11Processor.java:1041)
	at org.apache.coyote.AbstractProtocol$AbstractConnectionHandler.process(AbstractProtocol.java:603)
	at org.apache.tomcat.util.net.JIoEndpoint$SocketProcessor.run(JIoEndpoint.java:310)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at java.lang.Thread.run(Thread.java:748)
Caused by: com.alibaba.dubbo.remoting.TimeoutException: Waiting server-side response timeout by scan timer. start time: 2019-12-24 20:10:53.921, end time: 2019-12-24 20:10:54.943, client elapsed: 42 ms, server elapsed: 979 ms, timeout: 1000 ms, request: Request [id=1, version=2.0.0, twoway=true, event=false, broken=false, data=RpcInvocation [methodName=findPopSchool, parameterTypes=[int, int, class java.lang.Integer], arguments=[1, 10, 1], attachments={path=com.xqx.ych.small.api.IOrganizationApi, interface=com.xqx.ych.small.api.IOrganizationApi, version=0.0.0}]], channel: /192.168.71.1:2576 -> /192.168.71.1:20887
	at com.alibaba.dubbo.remoting.exchange.support.DefaultFuture.returnFromResponse(DefaultFuture.java:188)
	at com.alibaba.dubbo.remoting.exchange.support.DefaultFuture.get(DefaultFuture.java:110)
	at com.alibaba.dubbo.remoting.exchange.support.DefaultFuture.get(DefaultFuture.java:84)
	at com.alibaba.dubbo.rpc.protocol.dubbo.DubboInvoker.doInvoke(DubboInvoker.java:96)
	... 55 more
[2019-12-25 19:00:36] [WARN ] -  [DUBBO] Although retry the method findPopSchool in the service com.xqx.ych.small.api.IOrganizationApi was successful by the provider 192.168.71.1:20887, but there have been failed providers [192.168.71.1:20887] (1/1) from the registry 192.168.71.13:2181 on the consumer 192.168.71.1 using the dubbo version 2.8.4. Last error is: Invoke remote method timeout. method: findPopSchool, provider: dubbo://192.168.71.1:20887/com.xqx.ych.small.api.IOrganizationApi?anyhost=true&application=UserController&check=false&dubbo=2.8.4&generic=false&interface=com.xqx.ych.small.api.IOrganizationApi&methods=findByOrgLevel,findOrgDynamic,findPopSchool,findByMajorId,findOrgIntroduce,findByIdAndMajor,findByOrgId&pid=13068&revision=1.0-SNAPSHOT&side=consumer&timestamp=1577271627229, cause: Waiting server-side response timeout. start time: 2019-12-25 19:00:34.763, end time: 2019-12-25 19:00:35.783, client elapsed: 18 ms, server elapsed: 1002 ms, timeout: 1000 ms, request: Request [id=1, version=2.0.0, twoway=true, event=false, broken=false, data=RpcInvocation [methodName=findPopSchool, parameterTypes=[int, int, class java.lang.Integer], arguments=[1, 10, 1], attachments={path=com.xqx.ych.small.api.IOrganizationApi, interface=com.xqx.ych.small.api.IOrganizationApi, version=0.0.0}]], channel: /192.168.71.1:11773 -> /192.168.71.1:20887, dubbo version: 2.8.4, current host: 192.168.71.1
com.alibaba.dubbo.rpc.RpcException: Invoke remote method timeout. method: findPopSchool, provider: dubbo://192.168.71.1:20887/com.xqx.ych.small.api.IOrganizationApi?anyhost=true&application=UserController&check=false&dubbo=2.8.4&generic=false&interface=com.xqx.ych.small.api.IOrganizationApi&methods=findByOrgLevel,findOrgDynamic,findPopSchool,findByMajorId,findOrgIntroduce,findByIdAndMajor,findByOrgId&pid=13068&revision=1.0-SNAPSHOT&side=consumer&timestamp=1577271627229, cause: Waiting server-side response timeout. start time: 2019-12-25 19:00:34.763, end time: 2019-12-25 19:00:35.783, client elapsed: 18 ms, server elapsed: 1002 ms, timeout: 1000 ms, request: Request [id=1, version=2.0.0, twoway=true, event=false, broken=false, data=RpcInvocation [methodName=findPopSchool, parameterTypes=[int, int, class java.lang.Integer], arguments=[1, 10, 1], attachments={path=com.xqx.ych.small.api.IOrganizationApi, interface=com.xqx.ych.small.api.IOrganizationApi, version=0.0.0}]], channel: /192.168.71.1:11773 -> /192.168.71.1:20887
	at com.alibaba.dubbo.rpc.protocol.dubbo.DubboInvoker.doInvoke(DubboInvoker.java:99)
	at com.alibaba.dubbo.rpc.protocol.AbstractInvoker.invoke(AbstractInvoker.java:144)
	at com.alibaba.dubbo.rpc.listener.ListenerInvokerWrapper.invoke(ListenerInvokerWrapper.java:74)
	at com.alibaba.dubbo.monitor.support.MonitorFilter.invoke(MonitorFilter.java:75)
	at com.alibaba.dubbo.rpc.protocol.ProtocolFilterWrapper$1.invoke(ProtocolFilterWrapper.java:91)
	at com.alibaba.dubbo.rpc.protocol.dubbo.filter.FutureFilter.invoke(FutureFilter.java:53)
	at com.alibaba.dubbo.rpc.protocol.ProtocolFilterWrapper$1.invoke(ProtocolFilterWrapper.java:91)
	at com.alibaba.dubbo.rpc.filter.ConsumerContextFilter.invoke(ConsumerContextFilter.java:48)
	at com.alibaba.dubbo.rpc.protocol.ProtocolFilterWrapper$1.invoke(ProtocolFilterWrapper.java:91)
	at com.alibaba.dubbo.rpc.protocol.InvokerWrapper.invoke(InvokerWrapper.java:53)
	at com.alibaba.dubbo.rpc.cluster.support.FailoverClusterInvoker.doInvoke(FailoverClusterInvoker.java:77)
	at com.alibaba.dubbo.rpc.cluster.support.AbstractClusterInvoker.invoke(AbstractClusterInvoker.java:227)
	at com.alibaba.dubbo.rpc.cluster.support.wrapper.MockClusterInvoker.invoke(MockClusterInvoker.java:72)
	at com.alibaba.dubbo.rpc.proxy.InvokerInvocationHandler.invoke(InvokerInvocationHandler.java:52)
	at com.alibaba.dubbo.common.bytecode.proxy1.findPopSchool(proxy1.java)
	at com.xqx.small.web.OrganizationController.findPopSchool(OrganizationController.java:51)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.springframework.web.method.support.InvocableHandlerMethod.doInvoke(InvocableHandlerMethod.java:209)
	at org.springframework.web.method.support.InvocableHandlerMethod.invokeForRequest(InvocableHandlerMethod.java:136)
	at org.springframework.web.servlet.mvc.method.annotation.ServletInvocableHandlerMethod.invokeAndHandle(ServletInvocableHandlerMethod.java:102)
	at org.springframework.web.servlet.mvc.method.annotation.RequestMappingHandlerAdapter.invokeHandlerMethod(RequestMappingHandlerAdapter.java:894)
	at org.springframework.web.servlet.mvc.method.annotation.RequestMappingHandlerAdapter.handleInternal(RequestMappingHandlerAdapter.java:800)
	at org.springframework.web.servlet.mvc.method.AbstractHandlerMethodAdapter.handle(AbstractHandlerMethodAdapter.java:87)
	at org.springframework.web.servlet.DispatcherServlet.doDispatch(DispatcherServlet.java:991)
	at org.springframework.web.servlet.DispatcherServlet.doService(DispatcherServlet.java:925)
	at org.springframework.web.servlet.FrameworkServlet.processRequest(FrameworkServlet.java:981)
	at org.springframework.web.servlet.FrameworkServlet.doGet(FrameworkServlet.java:873)
	at javax.servlet.http.HttpServlet.service(HttpServlet.java:621)
	at org.springframework.web.servlet.FrameworkServlet.service(FrameworkServlet.java:858)
	at javax.servlet.http.HttpServlet.service(HttpServlet.java:728)
	at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:305)
	at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:210)
	at org.apache.tomcat.websocket.server.WsFilter.doFilter(WsFilter.java:51)
	at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:243)
	at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:210)
	at org.springframework.web.filter.CharacterEncodingFilter.doFilterInternal(CharacterEncodingFilter.java:200)
	at org.springframework.web.filter.OncePerRequestFilter.doFilter(OncePerRequestFilter.java:107)
	at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:243)
	at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:210)
	at org.apache.catalina.core.StandardWrapperValve.invoke(StandardWrapperValve.java:222)
	at org.apache.catalina.core.StandardContextValve.invoke(StandardContextValve.java:123)
	at org.apache.catalina.authenticator.AuthenticatorBase.invoke(AuthenticatorBase.java:502)
	at org.apache.catalina.core.StandardHostValve.invoke(StandardHostValve.java:171)
	at org.apache.catalina.valves.ErrorReportValve.invoke(ErrorReportValve.java:100)
	at org.apache.catalina.valves.AccessLogValve.invoke(AccessLogValve.java:953)
	at org.apache.catalina.core.StandardEngineValve.invoke(StandardEngineValve.java:118)
	at org.apache.catalina.connector.CoyoteAdapter.service(CoyoteAdapter.java:408)
	at org.apache.coyote.http11.AbstractHttp11Processor.process(AbstractHttp11Processor.java:1041)
	at org.apache.coyote.AbstractProtocol$AbstractConnectionHandler.process(AbstractProtocol.java:603)
	at org.apache.tomcat.util.net.JIoEndpoint$SocketProcessor.run(JIoEndpoint.java:310)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at java.lang.Thread.run(Thread.java:748)
Caused by: com.alibaba.dubbo.remoting.TimeoutException: Waiting server-side response timeout. start time: 2019-12-25 19:00:34.763, end time: 2019-12-25 19:00:35.783, client elapsed: 18 ms, server elapsed: 1002 ms, timeout: 1000 ms, request: Request [id=1, version=2.0.0, twoway=true, event=false, broken=false, data=RpcInvocation [methodName=findPopSchool, parameterTypes=[int, int, class java.lang.Integer], arguments=[1, 10, 1], attachments={path=com.xqx.ych.small.api.IOrganizationApi, interface=com.xqx.ych.small.api.IOrganizationApi, version=0.0.0}]], channel: /192.168.71.1:11773 -> /192.168.71.1:20887
	at com.alibaba.dubbo.remoting.exchange.support.DefaultFuture.get(DefaultFuture.java:107)
	at com.alibaba.dubbo.remoting.exchange.support.DefaultFuture.get(DefaultFuture.java:84)
	at com.alibaba.dubbo.rpc.protocol.dubbo.DubboInvoker.doInvoke(DubboInvoker.java:96)
	... 55 more
[2019-12-25 19:00:36] [WARN ] -  [DUBBO] The timeout response finally returned at 2019-12-25 19:00:36.580, response Response [id=1, version=null, status=20, event=false, error=null, result=RpcResult [result=PageRes(pageNum=1, pageSize=10, startRow=0, endRow=10, total=18, pages=2, count=true, list=[PopSchool(id=1, name=湖南大学, coverImg=https://kzych.oss-cn-qingdao.aliyuncs.com/kzych-dir/hndx_01.jpg), PopSchool(id=2, name=中南大学, coverImg=https://kzych.oss-cn-qingdao.aliyuncs.com/kzych-dir/hndx_01.jpg), PopSchool(id=3, name=湘潭大学, coverImg=https://kzych.oss-cn-qingdao.aliyuncs.com/kzych-dir/hndx_01.jpg), PopSchool(id=4, name=湖南农业大学, coverImg=https://kzych.oss-cn-qingdao.aliyuncs.com/kzych-dir/hndx_01.jpg), PopSchool(id=5, name=长沙理工大学, coverImg=https://kzych.oss-cn-qingdao.aliyuncs.com/kzych-dir/hndx_01.jpg), PopSchool(id=6, name=湖南工商大学, coverImg=https://kzych.oss-cn-qingdao.aliyuncs.com/kzych-dir/hndx_01.jpg), PopSchool(id=7, name=湖南理工大学, coverImg=https://kzych.oss-cn-qingdao.aliyuncs.com/kzych-dir/hndx_01.jpg), PopSchool(id=8, name=湖南师范大学, coverImg=https://kzych.oss-cn-qingdao.aliyuncs.com/kzych-dir/hndx_01.jpg), PopSchool(id=9, name=湖南女子大学, coverImg=https://kzych.oss-cn-qingdao.aliyuncs.com/kzych-dir/hndx_01.jpg), PopSchool(id=10, name=湖南工业大学, coverImg=https://kzych.oss-cn-qingdao.aliyuncs.com/kzych-dir/hndx_01.jpg)]), exception=null]], channel: /192.168.71.1:11773 -> /192.168.71.1:20887, dubbo version: 2.8.4, current host: 192.168.71.1
[2019-12-25 19:02:45] [WARN ] -  [DUBBO] client reconnect to 192.168.71.1:20887 find error . url: dubbo://192.168.71.1:20887/com.xqx.ych.small.api.IOrganizationApi?anyhost=true&application=UserController&check=false&codec=dubbo&dubbo=2.8.4&generic=false&heartbeat=60000&interface=com.xqx.ych.small.api.IOrganizationApi&methods=findByOrgLevel,findOrgDynamic,findPopSchool,findByMajorId,findOrgIntroduce,findByIdAndMajor,findByOrgId&pid=13068&revision=1.0-SNAPSHOT&side=consumer&timestamp=1577271627229, dubbo version: 2.8.4, current host: 192.168.71.1
com.alibaba.dubbo.remoting.RemotingException: client(url: dubbo://192.168.71.1:20887/com.xqx.ych.small.api.IOrganizationApi?anyhost=true&application=UserController&check=false&codec=dubbo&dubbo=2.8.4&generic=false&heartbeat=60000&interface=com.xqx.ych.small.api.IOrganizationApi&methods=findByOrgLevel,findOrgDynamic,findPopSchool,findByMajorId,findOrgIntroduce,findByIdAndMajor,findByOrgId&pid=13068&revision=1.0-SNAPSHOT&side=consumer&timestamp=1577271627229) failed to connect to server /192.168.71.1:20887, error message is:connection timed out: /192.168.71.1:20887
	at com.alibaba.dubbo.remoting.transport.netty.NettyClient.doConnect(NettyClient.java:142)
	at com.alibaba.dubbo.remoting.transport.AbstractClient.connect(AbstractClient.java:280)
	at com.alibaba.dubbo.remoting.transport.AbstractClient$1.run(AbstractClient.java:145)
	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511)
	at java.util.concurrent.FutureTask.runAndReset(FutureTask.java:308)
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.access$301(ScheduledThreadPoolExecutor.java:180)
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(ScheduledThreadPoolExecutor.java:294)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at java.lang.Thread.run(Thread.java:748)
Caused by: org.jboss.netty.channel.ConnectTimeoutException: connection timed out: /192.168.71.1:20887
	at org.jboss.netty.channel.socket.nio.NioClientBoss.processConnectTimeout(NioClientBoss.java:137)
	at org.jboss.netty.channel.socket.nio.NioClientBoss.process(NioClientBoss.java:83)
	at org.jboss.netty.channel.socket.nio.AbstractNioSelector.run(AbstractNioSelector.java:312)
	at org.jboss.netty.channel.socket.nio.NioClientBoss.run(NioClientBoss.java:42)
	... 3 more
[2019-12-25 19:02:50] [WARN ] -  [DUBBO] client reconnect to 192.168.71.1:20889 find error . url: dubbo://192.168.71.1:20889/com.xqx.ych.small.api.UserApi?anyhost=true&application=UserController&check=false&codec=dubbo&dubbo=2.8.4&generic=false&heartbeat=60000&interface=com.xqx.ych.small.api.UserApi&methods=register&pid=13068&revision=1.0-SNAPSHOT&side=consumer&timestamp=1577271630115, dubbo version: 2.8.4, current host: 192.168.71.1
com.alibaba.dubbo.remoting.RemotingException: client(url: dubbo://192.168.71.1:20889/com.xqx.ych.small.api.UserApi?anyhost=true&application=UserController&check=false&codec=dubbo&dubbo=2.8.4&generic=false&heartbeat=60000&interface=com.xqx.ych.small.api.UserApi&methods=register&pid=13068&revision=1.0-SNAPSHOT&side=consumer&timestamp=1577271630115) failed to connect to server /192.168.71.1:20889, error message is:connection timed out: /192.168.71.1:20889
	at com.alibaba.dubbo.remoting.transport.netty.NettyClient.doConnect(NettyClient.java:142)
	at com.alibaba.dubbo.remoting.transport.AbstractClient.connect(AbstractClient.java:280)
	at com.alibaba.dubbo.remoting.transport.AbstractClient$1.run(AbstractClient.java:145)
	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511)
	at java.util.concurrent.FutureTask.runAndReset(FutureTask.java:308)
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.access$301(ScheduledThreadPoolExecutor.java:180)
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(ScheduledThreadPoolExecutor.java:294)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at java.lang.Thread.run(Thread.java:748)
Caused by: org.jboss.netty.channel.ConnectTimeoutException: connection timed out: /192.168.71.1:20889
	at org.jboss.netty.channel.socket.nio.NioClientBoss.processConnectTimeout(NioClientBoss.java:137)
	at org.jboss.netty.channel.socket.nio.NioClientBoss.process(NioClientBoss.java:83)
	at org.jboss.netty.channel.socket.nio.AbstractNioSelector.run(AbstractNioSelector.java:312)
	at org.jboss.netty.channel.socket.nio.NioClientBoss.run(NioClientBoss.java:42)
	... 3 more
[2019-12-26 14:05:17] [WARN ] -  [DUBBO] client reconnect to 192.168.71.1:20887 find error . url: dubbo://192.168.71.1:20887/com.xqx.ych.small.api.IOrganizationApi?anyhost=true&application=UserController&check=false&codec=dubbo&default.retries=0&default.timeout=5000&dubbo=2.8.4&generic=false&heartbeat=60000&interface=com.xqx.ych.small.api.IOrganizationApi&methods=findByOrgLevel,findOrgDynamic,findByOrgMajorIdAndUserId,findPopSchool,findByMajorId,findOrgIntroduce,findByIdAndMajor,findByOrgId&pid=20172&revision=1.0-SNAPSHOT&side=consumer&timestamp=1577339916683, dubbo version: 2.8.4, current host: 192.168.71.1
com.alibaba.dubbo.remoting.RemotingException: client(url: dubbo://192.168.71.1:20887/com.xqx.ych.small.api.IOrganizationApi?anyhost=true&application=UserController&check=false&codec=dubbo&default.retries=0&default.timeout=5000&dubbo=2.8.4&generic=false&heartbeat=60000&interface=com.xqx.ych.small.api.IOrganizationApi&methods=findByOrgLevel,findOrgDynamic,findByOrgMajorIdAndUserId,findPopSchool,findByMajorId,findOrgIntroduce,findByIdAndMajor,findByOrgId&pid=20172&revision=1.0-SNAPSHOT&side=consumer&timestamp=1577339916683) failed to connect to server /192.168.71.1:20887, error message is:Connection refused: no further information: /192.168.71.1:20887
	at com.alibaba.dubbo.remoting.transport.netty.NettyClient.doConnect(NettyClient.java:142)
	at com.alibaba.dubbo.remoting.transport.AbstractClient.connect(AbstractClient.java:280)
	at com.alibaba.dubbo.remoting.transport.AbstractClient$1.run(AbstractClient.java:145)
	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511)
	at java.util.concurrent.FutureTask.runAndReset(FutureTask.java:308)
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.access$301(ScheduledThreadPoolExecutor.java:180)
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(ScheduledThreadPoolExecutor.java:294)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at java.lang.Thread.run(Thread.java:748)
Caused by: java.net.ConnectException: Connection refused: no further information: /192.168.71.1:20887
	at sun.nio.ch.SocketChannelImpl.checkConnect(Native Method)
	at sun.nio.ch.SocketChannelImpl.finishConnect(SocketChannelImpl.java:717)
	at org.jboss.netty.channel.socket.nio.NioClientBoss.connect(NioClientBoss.java:150)
	at org.jboss.netty.channel.socket.nio.NioClientBoss.processSelectedKeys(NioClientBoss.java:105)
	at org.jboss.netty.channel.socket.nio.NioClientBoss.process(NioClientBoss.java:79)
	at org.jboss.netty.channel.socket.nio.AbstractNioSelector.run(AbstractNioSelector.java:312)
	at org.jboss.netty.channel.socket.nio.NioClientBoss.run(NioClientBoss.java:42)
	... 3 more
[2019-12-26 14:08:24] [WARN ] -  [DUBBO] client reconnect to 192.168.71.1:20887 find error . url: dubbo://192.168.71.1:20887/com.xqx.ych.small.api.IOrganizationApi?anyhost=true&application=UserController&check=false&codec=dubbo&default.retries=0&default.timeout=5000&dubbo=2.8.4&generic=false&heartbeat=60000&interface=com.xqx.ych.small.api.IOrganizationApi&methods=findByOrgLevel,findOrgDynamic,findByOrgMajorIdAndUserId,findPopSchool,findOrgIntroduce,findByMajorId,findByIdAndMajor,findByOrgId&pid=12716&revision=1.0-SNAPSHOT&side=consumer&timestamp=1577340374109, dubbo version: 2.8.4, current host: 192.168.71.1
com.alibaba.dubbo.remoting.RemotingException: client(url: dubbo://192.168.71.1:20887/com.xqx.ych.small.api.IOrganizationApi?anyhost=true&application=UserController&check=false&codec=dubbo&default.retries=0&default.timeout=5000&dubbo=2.8.4&generic=false&heartbeat=60000&interface=com.xqx.ych.small.api.IOrganizationApi&methods=findByOrgLevel,findOrgDynamic,findByOrgMajorIdAndUserId,findPopSchool,findOrgIntroduce,findByMajorId,findByIdAndMajor,findByOrgId&pid=12716&revision=1.0-SNAPSHOT&side=consumer&timestamp=1577340374109) failed to connect to server /192.168.71.1:20887, error message is:Connection refused: no further information: /192.168.71.1:20887
	at com.alibaba.dubbo.remoting.transport.netty.NettyClient.doConnect(NettyClient.java:142)
	at com.alibaba.dubbo.remoting.transport.AbstractClient.connect(AbstractClient.java:280)
	at com.alibaba.dubbo.remoting.transport.AbstractClient$1.run(AbstractClient.java:145)
	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511)
	at java.util.concurrent.FutureTask.runAndReset(FutureTask.java:308)
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.access$301(ScheduledThreadPoolExecutor.java:180)
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(ScheduledThreadPoolExecutor.java:294)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at java.lang.Thread.run(Thread.java:748)
Caused by: java.net.ConnectException: Connection refused: no further information: /192.168.71.1:20887
	at sun.nio.ch.SocketChannelImpl.checkConnect(Native Method)
	at sun.nio.ch.SocketChannelImpl.finishConnect(SocketChannelImpl.java:717)
	at org.jboss.netty.channel.socket.nio.NioClientBoss.connect(NioClientBoss.java:150)
	at org.jboss.netty.channel.socket.nio.NioClientBoss.processSelectedKeys(NioClientBoss.java:105)
	at org.jboss.netty.channel.socket.nio.NioClientBoss.process(NioClientBoss.java:79)
	at org.jboss.netty.channel.socket.nio.AbstractNioSelector.run(AbstractNioSelector.java:312)
	at org.jboss.netty.channel.socket.nio.NioClientBoss.run(NioClientBoss.java:42)
	... 3 more
[2019-12-26 14:17:34] [WARN ] -  [DUBBO] client reconnect to 192.168.71.1:20887 find error . url: dubbo://192.168.71.1:20887/com.xqx.ych.small.api.IOrganizationApi?anyhost=true&application=UserController&check=false&codec=dubbo&default.retries=0&default.timeout=5000&dubbo=2.8.4&generic=false&heartbeat=60000&interface=com.xqx.ych.small.api.IOrganizationApi&methods=findByOrgLevel,findOrgDynamic,findByOrgMajorIdAndUserId,findPopSchool,findOrgIntroduce,findByMajorId,findByIdAndMajor,findByOrgId&pid=12716&revision=1.0-SNAPSHOT&side=consumer&timestamp=1577340374109, dubbo version: 2.8.4, current host: 192.168.71.1
com.alibaba.dubbo.remoting.RemotingException: client(url: dubbo://192.168.71.1:20887/com.xqx.ych.small.api.IOrganizationApi?anyhost=true&application=UserController&check=false&codec=dubbo&default.retries=0&default.timeout=5000&dubbo=2.8.4&generic=false&heartbeat=60000&interface=com.xqx.ych.small.api.IOrganizationApi&methods=findByOrgLevel,findOrgDynamic,findByOrgMajorIdAndUserId,findPopSchool,findOrgIntroduce,findByMajorId,findByIdAndMajor,findByOrgId&pid=12716&revision=1.0-SNAPSHOT&side=consumer&timestamp=1577340374109) failed to connect to server /192.168.71.1:20887, error message is:Connection refused: no further information: /192.168.71.1:20887
	at com.alibaba.dubbo.remoting.transport.netty.NettyClient.doConnect(NettyClient.java:142)
	at com.alibaba.dubbo.remoting.transport.AbstractClient.connect(AbstractClient.java:280)
	at com.alibaba.dubbo.remoting.transport.AbstractClient$1.run(AbstractClient.java:145)
	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511)
	at java.util.concurrent.FutureTask.runAndReset(FutureTask.java:308)
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.access$301(ScheduledThreadPoolExecutor.java:180)
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(ScheduledThreadPoolExecutor.java:294)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at java.lang.Thread.run(Thread.java:748)
Caused by: java.net.ConnectException: Connection refused: no further information: /192.168.71.1:20887
	at sun.nio.ch.SocketChannelImpl.checkConnect(Native Method)
	at sun.nio.ch.SocketChannelImpl.finishConnect(SocketChannelImpl.java:717)
	at org.jboss.netty.channel.socket.nio.NioClientBoss.connect(NioClientBoss.java:150)
	at org.jboss.netty.channel.socket.nio.NioClientBoss.processSelectedKeys(NioClientBoss.java:105)
	at org.jboss.netty.channel.socket.nio.NioClientBoss.process(NioClientBoss.java:79)
	at org.jboss.netty.channel.socket.nio.AbstractNioSelector.run(AbstractNioSelector.java:312)
	at org.jboss.netty.channel.socket.nio.NioClientBoss.run(NioClientBoss.java:42)
	... 3 more
[2019-12-26 14:17:36] [WARN ] -  [DUBBO] client reconnect to 192.168.71.1:20886 find error . url: dubbo://192.168.71.1:20886/com.xqx.ych.small.api.IMajorApi?anyhost=true&application=UserController&check=false&codec=dubbo&default.retries=0&default.timeout=5000&dubbo=2.8.4&generic=false&heartbeat=60000&interface=com.xqx.ych.small.api.IMajorApi&methods=findAllMajor,reduceOrgMajorNumber,queryOrgMajorById,findByCateId,findPopMajor&pid=12716&revision=1.0-SNAPSHOT&side=consumer&timestamp=1577340373522, dubbo version: 2.8.4, current host: 192.168.71.1
com.alibaba.dubbo.remoting.RemotingException: client(url: dubbo://192.168.71.1:20886/com.xqx.ych.small.api.IMajorApi?anyhost=true&application=UserController&check=false&codec=dubbo&default.retries=0&default.timeout=5000&dubbo=2.8.4&generic=false&heartbeat=60000&interface=com.xqx.ych.small.api.IMajorApi&methods=findAllMajor,reduceOrgMajorNumber,queryOrgMajorById,findByCateId,findPopMajor&pid=12716&revision=1.0-SNAPSHOT&side=consumer&timestamp=1577340373522) failed to connect to server /192.168.71.1:20886, error message is:Connection refused: no further information: /192.168.71.1:20886
	at com.alibaba.dubbo.remoting.transport.netty.NettyClient.doConnect(NettyClient.java:142)
	at com.alibaba.dubbo.remoting.transport.AbstractClient.connect(AbstractClient.java:280)
	at com.alibaba.dubbo.remoting.transport.AbstractClient$1.run(AbstractClient.java:145)
	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511)
	at java.util.concurrent.FutureTask.runAndReset(FutureTask.java:308)
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.access$301(ScheduledThreadPoolExecutor.java:180)
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(ScheduledThreadPoolExecutor.java:294)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at java.lang.Thread.run(Thread.java:748)
Caused by: java.net.ConnectException: Connection refused: no further information: /192.168.71.1:20886
	at sun.nio.ch.SocketChannelImpl.checkConnect(Native Method)
	at sun.nio.ch.SocketChannelImpl.finishConnect(SocketChannelImpl.java:717)
	at org.jboss.netty.channel.socket.nio.NioClientBoss.connect(NioClientBoss.java:150)
	at org.jboss.netty.channel.socket.nio.NioClientBoss.processSelectedKeys(NioClientBoss.java:105)
	at org.jboss.netty.channel.socket.nio.NioClientBoss.process(NioClientBoss.java:79)
	at org.jboss.netty.channel.socket.nio.AbstractNioSelector.run(AbstractNioSelector.java:312)
	at org.jboss.netty.channel.socket.nio.NioClientBoss.run(NioClientBoss.java:42)
	... 3 more
[2019-12-26 14:17:47] [WARN ] -  [DUBBO] client reconnect to 192.168.71.1:20889 find error . url: dubbo://192.168.71.1:20889/com.xqx.ych.small.api.UserApi?anyhost=true&application=UserController&check=false&codec=dubbo&default.retries=0&default.timeout=5000&dubbo=2.8.4&generic=false&heartbeat=60000&interface=com.xqx.ych.small.api.UserApi&methods=register&pid=12716&revision=1.0-SNAPSHOT&side=consumer&timestamp=1577340374243, dubbo version: 2.8.4, current host: 192.168.71.1
com.alibaba.dubbo.remoting.RemotingException: client(url: dubbo://192.168.71.1:20889/com.xqx.ych.small.api.UserApi?anyhost=true&application=UserController&check=false&codec=dubbo&default.retries=0&default.timeout=5000&dubbo=2.8.4&generic=false&heartbeat=60000&interface=com.xqx.ych.small.api.UserApi&methods=register&pid=12716&revision=1.0-SNAPSHOT&side=consumer&timestamp=1577340374243) failed to connect to server /192.168.71.1:20889, error message is:Connection refused: no further information: /192.168.71.1:20889
	at com.alibaba.dubbo.remoting.transport.netty.NettyClient.doConnect(NettyClient.java:142)
	at com.alibaba.dubbo.remoting.transport.AbstractClient.connect(AbstractClient.java:280)
	at com.alibaba.dubbo.remoting.transport.AbstractClient$1.run(AbstractClient.java:145)
	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511)
	at java.util.concurrent.FutureTask.runAndReset(FutureTask.java:308)
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.access$301(ScheduledThreadPoolExecutor.java:180)
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(ScheduledThreadPoolExecutor.java:294)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at java.lang.Thread.run(Thread.java:748)
Caused by: java.net.ConnectException: Connection refused: no further information: /192.168.71.1:20889
	at sun.nio.ch.SocketChannelImpl.checkConnect(Native Method)
	at sun.nio.ch.SocketChannelImpl.finishConnect(SocketChannelImpl.java:717)
	at org.jboss.netty.channel.socket.nio.NioClientBoss.connect(NioClientBoss.java:150)
	at org.jboss.netty.channel.socket.nio.NioClientBoss.processSelectedKeys(NioClientBoss.java:105)
	at org.jboss.netty.channel.socket.nio.NioClientBoss.process(NioClientBoss.java:79)
	at org.jboss.netty.channel.socket.nio.AbstractNioSelector.run(AbstractNioSelector.java:312)
	at org.jboss.netty.channel.socket.nio.NioClientBoss.run(NioClientBoss.java:42)
	... 3 more
[2019-12-26 14:21:48] [WARN ] -  [DUBBO] client reconnect to 192.168.71.1:20886 find error . url: dubbo://192.168.71.1:20886/com.xqx.ych.small.api.IMajorApi?anyhost=true&application=UserController&check=false&codec=dubbo&default.retries=0&default.timeout=5000&dubbo=2.8.4&generic=false&heartbeat=60000&interface=com.xqx.ych.small.api.IMajorApi&methods=findAllMajor,reduceOrgMajorNumber,queryOrgMajorById,findByCateId,findPopMajor&pid=16172&revision=1.0-SNAPSHOT&side=consumer&timestamp=1577341140948, dubbo version: 2.8.4, current host: 192.168.71.1
com.alibaba.dubbo.remoting.RemotingException: client(url: dubbo://192.168.71.1:20886/com.xqx.ych.small.api.IMajorApi?anyhost=true&application=UserController&check=false&codec=dubbo&default.retries=0&default.timeout=5000&dubbo=2.8.4&generic=false&heartbeat=60000&interface=com.xqx.ych.small.api.IMajorApi&methods=findAllMajor,reduceOrgMajorNumber,queryOrgMajorById,findByCateId,findPopMajor&pid=16172&revision=1.0-SNAPSHOT&side=consumer&timestamp=1577341140948) failed to connect to server /192.168.71.1:20886, error message is:Connection refused: no further information: /192.168.71.1:20886
	at com.alibaba.dubbo.remoting.transport.netty.NettyClient.doConnect(NettyClient.java:142)
	at com.alibaba.dubbo.remoting.transport.AbstractClient.connect(AbstractClient.java:280)
	at com.alibaba.dubbo.remoting.transport.AbstractClient$1.run(AbstractClient.java:145)
	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511)
	at java.util.concurrent.FutureTask.runAndReset(FutureTask.java:308)
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.access$301(ScheduledThreadPoolExecutor.java:180)
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(ScheduledThreadPoolExecutor.java:294)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at java.lang.Thread.run(Thread.java:748)
Caused by: java.net.ConnectException: Connection refused: no further information: /192.168.71.1:20886
	at sun.nio.ch.SocketChannelImpl.checkConnect(Native Method)
	at sun.nio.ch.SocketChannelImpl.finishConnect(SocketChannelImpl.java:717)
	at org.jboss.netty.channel.socket.nio.NioClientBoss.connect(NioClientBoss.java:150)
	at org.jboss.netty.channel.socket.nio.NioClientBoss.processSelectedKeys(NioClientBoss.java:105)
	at org.jboss.netty.channel.socket.nio.NioClientBoss.process(NioClientBoss.java:79)
	at org.jboss.netty.channel.socket.nio.AbstractNioSelector.run(AbstractNioSelector.java:312)
	at org.jboss.netty.channel.socket.nio.NioClientBoss.run(NioClientBoss.java:42)
	... 3 more
[2019-12-26 14:21:50] [WARN ] -  [DUBBO] client reconnect to 192.168.71.1:20889 find error . url: dubbo://192.168.71.1:20889/com.xqx.ych.small.api.UserApi?anyhost=true&application=UserController&check=false&codec=dubbo&default.retries=0&default.timeout=5000&dubbo=2.8.4&generic=false&heartbeat=60000&interface=com.xqx.ych.small.api.UserApi&methods=register&pid=16172&revision=1.0-SNAPSHOT&side=consumer&timestamp=1577341142337, dubbo version: 2.8.4, current host: 192.168.71.1
com.alibaba.dubbo.remoting.RemotingException: client(url: dubbo://192.168.71.1:20889/com.xqx.ych.small.api.UserApi?anyhost=true&application=UserController&check=false&codec=dubbo&default.retries=0&default.timeout=5000&dubbo=2.8.4&generic=false&heartbeat=60000&interface=com.xqx.ych.small.api.UserApi&methods=register&pid=16172&revision=1.0-SNAPSHOT&side=consumer&timestamp=1577341142337) failed to connect to server /192.168.71.1:20889, error message is:Connection refused: no further information: /192.168.71.1:20889
	at com.alibaba.dubbo.remoting.transport.netty.NettyClient.doConnect(NettyClient.java:142)
	at com.alibaba.dubbo.remoting.transport.AbstractClient.connect(AbstractClient.java:280)
	at com.alibaba.dubbo.remoting.transport.AbstractClient$1.run(AbstractClient.java:145)
	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511)
	at java.util.concurrent.FutureTask.runAndReset(FutureTask.java:308)
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.access$301(ScheduledThreadPoolExecutor.java:180)
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(ScheduledThreadPoolExecutor.java:294)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at java.lang.Thread.run(Thread.java:748)
Caused by: java.net.ConnectException: Connection refused: no further information: /192.168.71.1:20889
	at sun.nio.ch.SocketChannelImpl.checkConnect(Native Method)
	at sun.nio.ch.SocketChannelImpl.finishConnect(SocketChannelImpl.java:717)
	at org.jboss.netty.channel.socket.nio.NioClientBoss.connect(NioClientBoss.java:150)
	at org.jboss.netty.channel.socket.nio.NioClientBoss.processSelectedKeys(NioClientBoss.java:105)
	at org.jboss.netty.channel.socket.nio.NioClientBoss.process(NioClientBoss.java:79)
	at org.jboss.netty.channel.socket.nio.AbstractNioSelector.run(AbstractNioSelector.java:312)
	at org.jboss.netty.channel.socket.nio.NioClientBoss.run(NioClientBoss.java:42)
	... 3 more
[2019-12-26 14:25:42] [WARN ] -  [DUBBO] client reconnect to 192.168.71.1:20887 find error . url: dubbo://192.168.71.1:20887/com.xqx.ych.small.api.IOrganizationApi?anyhost=true&application=UserController&check=false&codec=dubbo&default.retries=0&default.timeout=5000&dubbo=2.8.4&generic=false&heartbeat=60000&interface=com.xqx.ych.small.api.IOrganizationApi&methods=findByOrgLevel,findOrgDynamic,findByOrgMajorIdAndUserId,findPopSchool,findByMajorId,findOrgIntroduce,findByIdAndMajor,findByOrgId&pid=2316&revision=1.0-SNAPSHOT&side=consumer&timestamp=1577341430312, dubbo version: 2.8.4, current host: 192.168.71.1
com.alibaba.dubbo.remoting.RemotingException: client(url: dubbo://192.168.71.1:20887/com.xqx.ych.small.api.IOrganizationApi?anyhost=true&application=UserController&check=false&codec=dubbo&default.retries=0&default.timeout=5000&dubbo=2.8.4&generic=false&heartbeat=60000&interface=com.xqx.ych.small.api.IOrganizationApi&methods=findByOrgLevel,findOrgDynamic,findByOrgMajorIdAndUserId,findPopSchool,findByMajorId,findOrgIntroduce,findByIdAndMajor,findByOrgId&pid=2316&revision=1.0-SNAPSHOT&side=consumer&timestamp=1577341430312) failed to connect to server /192.168.71.1:20887, error message is:Connection refused: no further information: /192.168.71.1:20887
	at com.alibaba.dubbo.remoting.transport.netty.NettyClient.doConnect(NettyClient.java:142)
	at com.alibaba.dubbo.remoting.transport.AbstractClient.connect(AbstractClient.java:280)
	at com.alibaba.dubbo.remoting.transport.AbstractClient$1.run(AbstractClient.java:145)
	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511)
	at java.util.concurrent.FutureTask.runAndReset(FutureTask.java:308)
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.access$301(ScheduledThreadPoolExecutor.java:180)
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(ScheduledThreadPoolExecutor.java:294)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at java.lang.Thread.run(Thread.java:748)
Caused by: java.net.ConnectException: Connection refused: no further information: /192.168.71.1:20887
	at sun.nio.ch.SocketChannelImpl.checkConnect(Native Method)
	at sun.nio.ch.SocketChannelImpl.finishConnect(SocketChannelImpl.java:717)
	at org.jboss.netty.channel.socket.nio.NioClientBoss.connect(NioClientBoss.java:150)
	at org.jboss.netty.channel.socket.nio.NioClientBoss.processSelectedKeys(NioClientBoss.java:105)
	at org.jboss.netty.channel.socket.nio.NioClientBoss.process(NioClientBoss.java:79)
	at org.jboss.netty.channel.socket.nio.AbstractNioSelector.run(AbstractNioSelector.java:312)
	at org.jboss.netty.channel.socket.nio.NioClientBoss.run(NioClientBoss.java:42)
	... 3 more
[2019-12-26 14:25:44] [WARN ] -  [DUBBO] client reconnect to 192.168.71.1:20886 find error . url: dubbo://192.168.71.1:20886/com.xqx.ych.small.api.IMajorApi?anyhost=true&application=UserController&check=false&codec=dubbo&default.retries=0&default.timeout=5000&dubbo=2.8.4&generic=false&heartbeat=60000&interface=com.xqx.ych.small.api.IMajorApi&methods=findAllMajor,reduceOrgMajorNumber,queryOrgMajorById,findByCateId,findPopMajor&pid=2316&revision=1.0-SNAPSHOT&side=consumer&timestamp=1577341429612, dubbo version: 2.8.4, current host: 192.168.71.1
com.alibaba.dubbo.remoting.RemotingException: client(url: dubbo://192.168.71.1:20886/com.xqx.ych.small.api.IMajorApi?anyhost=true&application=UserController&check=false&codec=dubbo&default.retries=0&default.timeout=5000&dubbo=2.8.4&generic=false&heartbeat=60000&interface=com.xqx.ych.small.api.IMajorApi&methods=findAllMajor,reduceOrgMajorNumber,queryOrgMajorById,findByCateId,findPopMajor&pid=2316&revision=1.0-SNAPSHOT&side=consumer&timestamp=1577341429612) failed to connect to server /192.168.71.1:20886, error message is:Connection refused: no further information: /192.168.71.1:20886
	at com.alibaba.dubbo.remoting.transport.netty.NettyClient.doConnect(NettyClient.java:142)
	at com.alibaba.dubbo.remoting.transport.AbstractClient.connect(AbstractClient.java:280)
	at com.alibaba.dubbo.remoting.transport.AbstractClient$1.run(AbstractClient.java:145)
	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511)
	at java.util.concurrent.FutureTask.runAndReset(FutureTask.java:308)
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.access$301(ScheduledThreadPoolExecutor.java:180)
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(ScheduledThreadPoolExecutor.java:294)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at java.lang.Thread.run(Thread.java:748)
Caused by: java.net.ConnectException: Connection refused: no further information: /192.168.71.1:20886
	at sun.nio.ch.SocketChannelImpl.checkConnect(Native Method)
	at sun.nio.ch.SocketChannelImpl.finishConnect(SocketChannelImpl.java:717)
	at org.jboss.netty.channel.socket.nio.NioClientBoss.connect(NioClientBoss.java:150)
	at org.jboss.netty.channel.socket.nio.NioClientBoss.processSelectedKeys(NioClientBoss.java:105)
	at org.jboss.netty.channel.socket.nio.NioClientBoss.process(NioClientBoss.java:79)
	at org.jboss.netty.channel.socket.nio.AbstractNioSelector.run(AbstractNioSelector.java:312)
	at org.jboss.netty.channel.socket.nio.NioClientBoss.run(NioClientBoss.java:42)
	... 3 more
[2019-12-26 16:47:24] [WARN ] -  [DUBBO] client reconnect to 192.168.71.1:20887 find error . url: dubbo://192.168.71.1:20887/com.xqx.ych.small.api.IOrganizationApi?anyhost=true&application=UserController&check=false&codec=dubbo&default.retries=0&default.timeout=5000&dubbo=2.8.4&generic=false&heartbeat=60000&interface=com.xqx.ych.small.api.IOrganizationApi&methods=findByOrgLevel,findOrgDynamic,findByOrgMajorIdAndUserId,findPopSchool,findByMajorId,findOrgIntroduce,findByIdAndMajor,findByOrgId&pid=4904&revision=1.0-SNAPSHOT&side=consumer&timestamp=1577341595993, dubbo version: 2.8.4, current host: 192.168.71.1
com.alibaba.dubbo.remoting.RemotingException: client(url: dubbo://192.168.71.1:20887/com.xqx.ych.small.api.IOrganizationApi?anyhost=true&application=UserController&check=false&codec=dubbo&default.retries=0&default.timeout=5000&dubbo=2.8.4&generic=false&heartbeat=60000&interface=com.xqx.ych.small.api.IOrganizationApi&methods=findByOrgLevel,findOrgDynamic,findByOrgMajorIdAndUserId,findPopSchool,findByMajorId,findOrgIntroduce,findByIdAndMajor,findByOrgId&pid=4904&revision=1.0-SNAPSHOT&side=consumer&timestamp=1577341595993) failed to connect to server /192.168.71.1:20887, error message is:Connection refused: no further information: /192.168.71.1:20887
	at com.alibaba.dubbo.remoting.transport.netty.NettyClient.doConnect(NettyClient.java:142)
	at com.alibaba.dubbo.remoting.transport.AbstractClient.connect(AbstractClient.java:280)
	at com.alibaba.dubbo.remoting.transport.AbstractClient$1.run(AbstractClient.java:145)
	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511)
	at java.util.concurrent.FutureTask.runAndReset(FutureTask.java:308)
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.access$301(ScheduledThreadPoolExecutor.java:180)
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(ScheduledThreadPoolExecutor.java:294)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at java.lang.Thread.run(Thread.java:748)
Caused by: java.net.ConnectException: Connection refused: no further information: /192.168.71.1:20887
	at sun.nio.ch.SocketChannelImpl.checkConnect(Native Method)
	at sun.nio.ch.SocketChannelImpl.finishConnect(SocketChannelImpl.java:717)
	at org.jboss.netty.channel.socket.nio.NioClientBoss.connect(NioClientBoss.java:150)
	at org.jboss.netty.channel.socket.nio.NioClientBoss.processSelectedKeys(NioClientBoss.java:105)
	at org.jboss.netty.channel.socket.nio.NioClientBoss.process(NioClientBoss.java:79)
	at org.jboss.netty.channel.socket.nio.AbstractNioSelector.run(AbstractNioSelector.java:312)
	at org.jboss.netty.channel.socket.nio.NioClientBoss.run(NioClientBoss.java:42)
	... 3 more
[2019-12-26 16:47:34] [WARN ] -  [DUBBO] client reconnect to 192.168.71.1:20886 find error . url: dubbo://192.168.71.1:20886/com.xqx.ych.small.api.IMajorApi?anyhost=true&application=UserController&check=false&codec=dubbo&default.retries=0&default.timeout=5000&dubbo=2.8.4&generic=false&heartbeat=60000&interface=com.xqx.ych.small.api.IMajorApi&methods=findAllMajor,reduceOrgMajorNumber,queryOrgMajorById,findByCateId,findPopMajor&pid=4904&revision=1.0-SNAPSHOT&side=consumer&timestamp=1577341595524, dubbo version: 2.8.4, current host: 192.168.71.1
com.alibaba.dubbo.remoting.RemotingException: client(url: dubbo://192.168.71.1:20886/com.xqx.ych.small.api.IMajorApi?anyhost=true&application=UserController&check=false&codec=dubbo&default.retries=0&default.timeout=5000&dubbo=2.8.4&generic=false&heartbeat=60000&interface=com.xqx.ych.small.api.IMajorApi&methods=findAllMajor,reduceOrgMajorNumber,queryOrgMajorById,findByCateId,findPopMajor&pid=4904&revision=1.0-SNAPSHOT&side=consumer&timestamp=1577341595524) failed to connect to server /192.168.71.1:20886, error message is:Connection refused: no further information: /192.168.71.1:20886
	at com.alibaba.dubbo.remoting.transport.netty.NettyClient.doConnect(NettyClient.java:142)
	at com.alibaba.dubbo.remoting.transport.AbstractClient.connect(AbstractClient.java:280)
	at com.alibaba.dubbo.remoting.transport.AbstractClient$1.run(AbstractClient.java:145)
	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511)
	at java.util.concurrent.FutureTask.runAndReset(FutureTask.java:308)
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.access$301(ScheduledThreadPoolExecutor.java:180)
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(ScheduledThreadPoolExecutor.java:294)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at java.lang.Thread.run(Thread.java:748)
Caused by: java.net.ConnectException: Connection refused: no further information: /192.168.71.1:20886
	at sun.nio.ch.SocketChannelImpl.checkConnect(Native Method)
	at sun.nio.ch.SocketChannelImpl.finishConnect(SocketChannelImpl.java:717)
	at org.jboss.netty.channel.socket.nio.NioClientBoss.connect(NioClientBoss.java:150)
	at org.jboss.netty.channel.socket.nio.NioClientBoss.processSelectedKeys(NioClientBoss.java:105)
	at org.jboss.netty.channel.socket.nio.NioClientBoss.process(NioClientBoss.java:79)
	at org.jboss.netty.channel.socket.nio.AbstractNioSelector.run(AbstractNioSelector.java:312)
	at org.jboss.netty.channel.socket.nio.NioClientBoss.run(NioClientBoss.java:42)
	... 3 more
[2019-12-26 17:07:58] [WARN ] -  [DUBBO] client reconnect to 192.168.71.1:20886 find error . url: dubbo://192.168.71.1:20886/com.xqx.ych.small.api.IMajorApi?anyhost=true&application=UserController&check=false&codec=dubbo&default.retries=0&default.timeout=5000&dubbo=2.8.4&generic=false&heartbeat=60000&interface=com.xqx.ych.small.api.IMajorApi&methods=findAllMajor,reduceOrgMajorNumber,queryOrgMajorById,findByCateId,findPopMajor&pid=20524&revision=1.0-SNAPSHOT&side=consumer&timestamp=1577350554105, dubbo version: 2.8.4, current host: 192.168.71.1
com.alibaba.dubbo.remoting.RemotingException: client(url: dubbo://192.168.71.1:20886/com.xqx.ych.small.api.IMajorApi?anyhost=true&application=UserController&check=false&codec=dubbo&default.retries=0&default.timeout=5000&dubbo=2.8.4&generic=false&heartbeat=60000&interface=com.xqx.ych.small.api.IMajorApi&methods=findAllMajor,reduceOrgMajorNumber,queryOrgMajorById,findByCateId,findPopMajor&pid=20524&revision=1.0-SNAPSHOT&side=consumer&timestamp=1577350554105) failed to connect to server /192.168.71.1:20886, error message is:Connection refused: no further information: /192.168.71.1:20886
	at com.alibaba.dubbo.remoting.transport.netty.NettyClient.doConnect(NettyClient.java:142)
	at com.alibaba.dubbo.remoting.transport.AbstractClient.connect(AbstractClient.java:280)
	at com.alibaba.dubbo.remoting.transport.AbstractClient$1.run(AbstractClient.java:145)
	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511)
	at java.util.concurrent.FutureTask.runAndReset(FutureTask.java:308)
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.access$301(ScheduledThreadPoolExecutor.java:180)
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(ScheduledThreadPoolExecutor.java:294)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at java.lang.Thread.run(Thread.java:748)
Caused by: java.net.ConnectException: Connection refused: no further information: /192.168.71.1:20886
	at sun.nio.ch.SocketChannelImpl.checkConnect(Native Method)
	at sun.nio.ch.SocketChannelImpl.finishConnect(SocketChannelImpl.java:717)
	at org.jboss.netty.channel.socket.nio.NioClientBoss.connect(NioClientBoss.java:150)
	at org.jboss.netty.channel.socket.nio.NioClientBoss.processSelectedKeys(NioClientBoss.java:105)
	at org.jboss.netty.channel.socket.nio.NioClientBoss.process(NioClientBoss.java:79)
	at org.jboss.netty.channel.socket.nio.AbstractNioSelector.run(AbstractNioSelector.java:312)
	at org.jboss.netty.channel.socket.nio.NioClientBoss.run(NioClientBoss.java:42)
	... 3 more
[2019-12-26 19:13:20] [WARN ] -  [DUBBO] Failed to save registry store file, cause: Can not lock the registry cache file C:\Users\lxxqx\.dubbo\dubbo-registry-192.168.71.13.cache, ignore and retry later, maybe multi java process use the file, please config: dubbo.registry.file=xxx.properties, dubbo version: 2.8.4, current host: 192.168.71.1
java.io.IOException: Can not lock the registry cache file C:\Users\lxxqx\.dubbo\dubbo-registry-192.168.71.13.cache, ignore and retry later, maybe multi java process use the file, please config: dubbo.registry.file=xxx.properties
	at com.alibaba.dubbo.registry.support.AbstractRegistry.doSaveProperties(AbstractRegistry.java:193)
	at com.alibaba.dubbo.registry.support.AbstractRegistry$SaveProperties.run(AbstractRegistry.java:150)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at java.lang.Thread.run(Thread.java:748)
[2019-12-26 19:21:36] [WARN ] -  [DUBBO] Failed to save registry store file, cause: Can not lock the registry cache file C:\Users\lxxqx\.dubbo\dubbo-registry-192.168.71.13.cache, ignore and retry later, maybe multi java process use the file, please config: dubbo.registry.file=xxx.properties, dubbo version: 2.8.4, current host: 192.168.71.1
java.io.IOException: Can not lock the registry cache file C:\Users\lxxqx\.dubbo\dubbo-registry-192.168.71.13.cache, ignore and retry later, maybe multi java process use the file, please config: dubbo.registry.file=xxx.properties
	at com.alibaba.dubbo.registry.support.AbstractRegistry.doSaveProperties(AbstractRegistry.java:193)
	at com.alibaba.dubbo.registry.support.AbstractRegistry$SaveProperties.run(AbstractRegistry.java:150)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at java.lang.Thread.run(Thread.java:748)
[2019-12-26 19:55:02] [WARN ] -  [DUBBO] Failed to save registry store file, cause: Can not lock the registry cache file C:\Users\lxxqx\.dubbo\dubbo-registry-192.168.71.13.cache, ignore and retry later, maybe multi java process use the file, please config: dubbo.registry.file=xxx.properties, dubbo version: 2.8.4, current host: 192.168.71.1
java.io.IOException: Can not lock the registry cache file C:\Users\lxxqx\.dubbo\dubbo-registry-192.168.71.13.cache, ignore and retry later, maybe multi java process use the file, please config: dubbo.registry.file=xxx.properties
	at com.alibaba.dubbo.registry.support.AbstractRegistry.doSaveProperties(AbstractRegistry.java:193)
	at com.alibaba.dubbo.registry.support.AbstractRegistry$SaveProperties.run(AbstractRegistry.java:150)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at java.lang.Thread.run(Thread.java:748)
[2019-12-26 19:55:19] [WARN ] -  [DUBBO] Failed to save registry store file, cause: Can not lock the registry cache file C:\Users\lxxqx\.dubbo\dubbo-registry-192.168.71.13.cache, ignore and retry later, maybe multi java process use the file, please config: dubbo.registry.file=xxx.properties, dubbo version: 2.8.4, current host: 192.168.71.1
java.io.IOException: Can not lock the registry cache file C:\Users\lxxqx\.dubbo\dubbo-registry-192.168.71.13.cache, ignore and retry later, maybe multi java process use the file, please config: dubbo.registry.file=xxx.properties
	at com.alibaba.dubbo.registry.support.AbstractRegistry.doSaveProperties(AbstractRegistry.java:193)
	at com.alibaba.dubbo.registry.support.AbstractRegistry$SaveProperties.run(AbstractRegistry.java:150)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at java.lang.Thread.run(Thread.java:748)
[2019-12-26 20:19:53] [WARN ] -  [DUBBO] Failed to save registry store file, cause: Can not lock the registry cache file C:\Users\lxxqx\.dubbo\dubbo-registry-192.168.71.13.cache, ignore and retry later, maybe multi java process use the file, please config: dubbo.registry.file=xxx.properties, dubbo version: 2.8.4, current host: 192.168.71.1
java.io.IOException: Can not lock the registry cache file C:\Users\lxxqx\.dubbo\dubbo-registry-192.168.71.13.cache, ignore and retry later, maybe multi java process use the file, please config: dubbo.registry.file=xxx.properties
	at com.alibaba.dubbo.registry.support.AbstractRegistry.doSaveProperties(AbstractRegistry.java:193)
	at com.alibaba.dubbo.registry.support.AbstractRegistry$SaveProperties.run(AbstractRegistry.java:150)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at java.lang.Thread.run(Thread.java:748)
[2019-12-26 20:19:56] [WARN ] -  [DUBBO] Failed to save registry store file, cause: Can not lock the registry cache file C:\Users\lxxqx\.dubbo\dubbo-registry-192.168.71.13.cache, ignore and retry later, maybe multi java process use the file, please config: dubbo.registry.file=xxx.properties, dubbo version: 2.8.4, current host: 192.168.71.1
java.io.IOException: Can not lock the registry cache file C:\Users\lxxqx\.dubbo\dubbo-registry-192.168.71.13.cache, ignore and retry later, maybe multi java process use the file, please config: dubbo.registry.file=xxx.properties
	at com.alibaba.dubbo.registry.support.AbstractRegistry.doSaveProperties(AbstractRegistry.java:193)
	at com.alibaba.dubbo.registry.support.AbstractRegistry$SaveProperties.run(AbstractRegistry.java:150)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at java.lang.Thread.run(Thread.java:748)
[2019-12-26 20:19:56] [WARN ] -  [DUBBO] Failed to save registry store file, cause: Can not lock the registry cache file C:\Users\lxxqx\.dubbo\dubbo-registry-192.168.71.13.cache, ignore and retry later, maybe multi java process use the file, please config: dubbo.registry.file=xxx.properties, dubbo version: 2.8.4, current host: 192.168.71.1
java.io.IOException: Can not lock the registry cache file C:\Users\lxxqx\.dubbo\dubbo-registry-192.168.71.13.cache, ignore and retry later, maybe multi java process use the file, please config: dubbo.registry.file=xxx.properties
	at com.alibaba.dubbo.registry.support.AbstractRegistry.doSaveProperties(AbstractRegistry.java:193)
	at com.alibaba.dubbo.registry.support.AbstractRegistry$SaveProperties.run(AbstractRegistry.java:150)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at java.lang.Thread.run(Thread.java:748)
[2019-12-26 20:20:02] [WARN ] -  [DUBBO] Failed to save registry store file, cause: Can not lock the registry cache file C:\Users\lxxqx\.dubbo\dubbo-registry-192.168.71.13.cache, ignore and retry later, maybe multi java process use the file, please config: dubbo.registry.file=xxx.properties, dubbo version: 2.8.4, current host: 192.168.71.1
java.io.IOException: Can not lock the registry cache file C:\Users\lxxqx\.dubbo\dubbo-registry-192.168.71.13.cache, ignore and retry later, maybe multi java process use the file, please config: dubbo.registry.file=xxx.properties
	at com.alibaba.dubbo.registry.support.AbstractRegistry.doSaveProperties(AbstractRegistry.java:193)
	at com.alibaba.dubbo.registry.support.AbstractRegistry$SaveProperties.run(AbstractRegistry.java:150)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at java.lang.Thread.run(Thread.java:748)
[2019-12-26 20:20:02] [WARN ] -  [DUBBO] Failed to save registry store file, cause: Can not lock the registry cache file C:\Users\lxxqx\.dubbo\dubbo-registry-192.168.71.13.cache, ignore and retry later, maybe multi java process use the file, please config: dubbo.registry.file=xxx.properties, dubbo version: 2.8.4, current host: 192.168.71.1
java.io.IOException: Can not lock the registry cache file C:\Users\lxxqx\.dubbo\dubbo-registry-192.168.71.13.cache, ignore and retry later, maybe multi java process use the file, please config: dubbo.registry.file=xxx.properties
	at com.alibaba.dubbo.registry.support.AbstractRegistry.doSaveProperties(AbstractRegistry.java:193)
	at com.alibaba.dubbo.registry.support.AbstractRegistry$SaveProperties.run(AbstractRegistry.java:150)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at java.lang.Thread.run(Thread.java:748)
